README.markdown

```diff
@@ -1,20 +1,11 @@
 # About _gprof2dot_
```

Comment:
Add a comment with a reference to the original project here? Something like "this is a fork of [link to original] tailored to Yelp's need.